### ISSUE TITLE
Can't get the field to display

### ISSUE THREAD

**Issue by MikeHarrison** [AndrewHaine/statamic-cartographer#9]
Hi,

I am trialling Cartographer in a local Statamic install (downloaded a trial version, not purchased a license yet), and can't seem to get it to play ball. I have installed the Addon, and added my Mapbox API key in the settings. I have also added a "Map" block in my Replicator page builder.

However when I add a Map block I get the following message:

**Please enter your Mapbox access token in the addon settings.**

It is definitely in there, and is not restricted by URL or anything like that. Is there a step I have missed in setup?

Thanks for any pointers you can offer, looks like the perfect addon for what I need if I can get it working!

**Comment by MikeHarrison** [comment]
Update on this - it seems to be because it is in a Replicator. Outside of one everything works fine. This is the error I am getting in my JS console:

```
Uncaught (in promise) TypeError: Cannot read property 'lat' of undefined
    at CartographerFieldtype.getReplicatorPreviewText (fieldtype.js:1)
    at CartographerFieldtype.getReplicatorPreviewText (vendor.js?v=2.11.20:1)
    at bootstrap.js?v=2.11.20:1
    at Function.v.map.v.collect (vendor.js?v=2.11.20:1)
    at VueComponent.getCollapsedPreview (bootstrap.js?v=2.11.20:1)
    at VueComponent.getCollapsedPreview (vendor.js?v=2.11.20:1)
    at VueComponent.ready (bootstrap.js?v=2.11.20:1)
    at VueComponent.e._callHook (vendor.js?v=2.11.20:1)
    at VueComponent.t (vendor.js?v=2.11.20:1)
    at VueComponent.i (vendor.js?v=2.11.20:1)
```

**Comment by AndrewHaine** [comment]
Thanks for reporting, looks like no data for the map center is being passed to the function to generates the preview text, i'll get a fix together to account for when no data is available

**Comment by AndrewHaine** [comment]
Hey Mike, Just released 1.0.2 which should fix the issue you were seeing but please re-open if you have any issues!
